source/renderer/app/components/wallet/utxo/WalletUtxoDescription.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+import React from 'react';
+import { FormattedHTMLMessage } from 'react-intl';
+import { useDiscreetModeFeature } from '../../../features/discreet-mode';
+import type { ReactIntlMessage } from '../../../types/i18nTypes';
+
+type Props = {
+  description: ReactIntlMessage,
+  formattedWalletAmount: string,
+  walletUtxosAmount: number,
+};
+
+export function WalletUtxoDescription({
```

Comment:
i think this should be wrapped in mobx `observer`, otherwise it might not update correctly when switching the discreet mode on/off?